game_event/views.py

```diff
@@ -63,3 +65,52 @@ def game_events(request):
         game_events = game_events.filter(time__gte=current_date)
 
     return render(request, 'game-events.html', {'game_events': game_events})
+
+
+def game_event(request, id):
+    event = get_object_or_404(GameEvent, pk=id)
+    player = Player.objects.get(user=request.user)
+    event_players = GameEventPlayer.objects.filter(game_event=event)
+    event_players = [{'first_name': entry.player.user.first_name, 'last_name': entry.player.user.last_name,
+                     'brings_ball': entry.ball_responsible} for entry in event_players]
+    in_event = False
+    if GameEventPlayer.objects.filter(game_event=event, player=player).exists():
+        in_event = True
+    context = {'id': event.id, 'time': event.time, 'level_of_game': event.level_of_game, 'min_number_of_players':
+               event.min_number_of_players, 'max_number_of_players': event.max_number_of_players,
+               'court': event.court.city, 'neighborhood': event.court.neighborhood,
+               'ball_game': event.ball_game, 'in_event': in_event, 'event_players': event_players}
+
+    return render(request, 'game_event/game-event.html', context)
+
+
+def join_event(request, id):
+    get_object_or_404(GameEvent, pk=id)
+    return render(request, 'game_event/join-event.html', {'id': id})
+
+
+def remove_from_event(request, id):
+    event = get_object_or_404(GameEvent, id=id)
+    player = Player.objects.get(user=request.user)
+    player_in_event = GameEventPlayer.objects.get(game_event=event, player=player)
+    player_in_event.delete()
+    return render(request, 'game_event/result.html')
+
+
+def process_answer(request, id):
+    event = get_object_or_404(GameEvent, pk=id)
+    player = Player.objects.get(user=request.user)
+    selected_option = request.POST.get('answer')
+    if selected_option == 'yes':
```

Comment:
I think it would be easier to understand if you don't write the create method twice.
```

ball_responsible = selected_option == 'yes'
GameEventPlayer.objects.create(game_event=event, player=player, ball_responsible=ball_responsible)

```